frontend/lib/src/components/widgets/CameraInput/CameraInput.tsx

```diff
@@ -356,7 +356,7 @@ class CameraInput extends React.PureComponent<Props, State> {
     return (
       <StyledCameraInput
         width={width}
-        className="row-widget"
+        className="row-widget stCameraInput"
```

Comment:
Why add the class?

Reply:
Oh, yeah.. In a previous version I also added some consistency changes to classes & test IDs but I removed most of those changes again since its better to do that in a dedicated PR. Missed this one.